docs/custom-instrumentation.asciidoc

```diff
@@ -1,12 +1,21 @@
 [[instrumenting-custom-code]]
 === Instrumenting custom code
 
+[float]
+[[instrumenting-custom-code-spans]]
+==== Creating Additional Spans in a Transaction
+
 Elastic APM instruments a variety of libraries out of the box, but sometimes you
 need to know how long a specific function took or how often it gets
 called.
 
-You can apply the `@elasticapm.capture_span()` decorator to achieve exactly that.
-It can also be used as a context manager in the following way:
+Assuming you're using one of our <<set-up, supported frameworks>>, you can
+apply the `@elasticapm.capture_span()` decorator to achieve exactly that. (If
+you're not using a supported framework, see
+<<instrumenting-custom-code-transactions, Creating New Transactions>>)
+
+`elasticapm.capture_span` can be used either as a decorator, or as a context
```

Comment:
```suggestion
`elasticapm.capture_span` can be used either as a decorator or context
```

Reply:
On second thought, I think I like the `as a` here. Regardless, with correlative conjunctions (in this case, "either" ... "or") there's no need to use a comma before the second conjunction.

```suggestion
`elasticapm.capture_span` can be used either as a decorator or as a context
```